src/treemap/iter.rs

```diff
@@ -221,6 +221,50 @@ impl Extend<u64> for RoaringTreemap {
     }
 }
 
+impl RoaringTreemap {
+    /// Create the set from a sorted iterator.
+    /// The value **must** be strictly sorted.
```

Comment:
```suggestion
    /// Create the set from a sorted iterator. Values **must** be sorted.
```

Am I right here too? Can values be also equal?